src/styles/_catalog.scss

```diff
@@ -1,22 +1,26 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: $gap;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill, $width);
 
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
-    justify-content: center;
+    display: grid;
+    justify-items: center;
     align-items: center;
-    width: 200px;
+    width: $width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
   }
+}
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+@media (max-width: 488px) {
```

Comment:
Media queries
There should be four columns starting at 1024px
Now: U show 4 columns from 960px
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/86183007/187827117-04708b15-23b5-42b4-a80d-d7d3ede72e20.png">

There should be three columns starting at 768px
Now: U show 3 columns from 712px
<img width="765" alt="image" src="https://user-images.githubusercontent.com/86183007/187827381-bb7d76c3-b5d1-4f04-a49c-9aeb5429852f.png">

There should be two columns starting at 488px
Now: U show 2 columns from 464px
<img width="530" alt="image" src="https://user-images.githubusercontent.com/86183007/187827445-6a48036a-3037-4891-a675-235ae037652d.png">

There should be one column below 488px
And about this point, u show one column below 464px